install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/rsync-filter

```diff
@@ -0,0 +1,36 @@
+exclude config.py
+include *.py
+include COPYING
+include dictionaries/
+include dictionaries/*.txt
+include journalist_templates/
+include journalist_templates/*.html
+include management/
+include management/*.py
+include requirements/
+include requirements/securedrop-requirements.txt
+include source_templates/
+include source_templates/*.html
+include static/
+include static/fonts/
+include static/fonts/**
+include static/i/
+include static/i/**
+include static/i/font-awesome/
+include static/i/font-awesome/**
+include static/i/tipbox/
```

Comment:
Assuming you mean `static/i/tipbox/`, I see it on present development:

```
./securedrop/static/i/tipbox
./securedrop/static/i/tipbox/tipbox-hed-j-all.png
./securedrop/static/i/tipbox/tipbox-logo.png
./securedrop/static/i/tipbox/tipbox-hed-submit3.png
./securedrop/static/i/tipbox/tipbox-hed-user.png
./securedrop/static/i/tipbox/tipbox-hed-j-single.png
./securedrop/static/i/tipbox/tipbox-hed-submit1.png
```

Now as far as whether those includes are _used_, that's a separate matter! 

Will follow-up with some tests-related changes to deb packages build logic, since the additive approach I took wasn't quite additive enough. Resisting the urge to rsync the entire tests dir and stick with the selective approach.